docs/release-train-issue-template.md

```diff
@@ -0,0 +1,64 @@
+Release Akka Persistence R2DBC $VERSION$
+
+<!--
+# Release Train Issue Template for Akka Persistence R2DBC
+
+(Liberally copied and adopted from Scala itself https://github.com/scala/scala-dev/blob/b11cd2e4a4431de7867db6b39362bea8fa6650e7/notes/releases/template.md)
+
+For every release, use the `scripts/create-release-issue.sh` to make a copy of this file named after the release, and expand the variables.
+
+Variables to be expanded in this template:
+- $VERSION$=???
+
+Key links:
+  - akka/akka-persistence-r2dbc milestone: https://github.com/akka/akka-peristence-r2dbc/milestone/?
+-->
+
+### Cutting the release
+
+- [ ] Check that open PRs and issues assigned to the milestone are reasonable
+- [ ] For minor or major versions, update the Change date in the LICENSE file and update the `licenses` url in the build.
+- [ ] Create a new milestone for the [next version](https://github.com/akka/akka-persistence-r2dbc/milestones)
+- [ ] Close the [$VERSION$ milestone](https://github.com/akka/akka-persistence-r2dbc/milestones?direction=asc&sort=due_date)
+- [ ] Make sure all important PRs have been merged
```

Comment:
Here we could also add a Run of Fossa and to update the revision in Fossa.

Reply:
Or just take a look at the nightly fossa run and see that it is green? Update revision, I'm not quite sure what that means.

Reply:
It's scheduled once per week. Should typically be enough. I can mention that and mention that if there has been recent dependency changes it can be triggered manually.

Reply:
https://github.com/akka/akka-persistence-r2dbc/pull/291/commits/72981976efde1b92f879f8e408fbb92e1c4fffdb